Dockerfile

```diff
@@ -18,6 +18,8 @@ RUN : \
     "${ARCH:?ARCH is required to build}" \
     "${SSM_AGENT_VERSION:?SSM Agent version required to build}"
 
+RUN yum -y update
```

Comment:
I'd link this into the RUN command below, e.g. `RUN ... yum -y update && yum install -y amazon-ssm-agent.rpm`, to avoid creating another layer.